src/components/Header/DesktopMenuDropdown.js

```diff
@@ -0,0 +1,167 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import is from 'styled-is'
+import remcalc from 'remcalc'
+import Chevron from '../Common/Chevron'
+import { DesktopMenuItem, outlineStyles } from './elements'
+import Anchor from '../Common/Anchor'
+import theme from '../../utils/theme'
+
+const dropDownItemPadding = `padding: 10px 15px 14px 15px;`
+
+const DesktopMenuDropdownContainer = styled(DesktopMenuItem)`
+  position: relative;
+  transition: color ${props => props.theme.animations.fast} ease-in-out,
+    background ${props => props.theme.animations.fast} ease-in-out;
+  cursor: pointer;
+  background: transparent;
+
+  ${is('darkTheme')`
+    color: ${props => props.theme.colors.white};
+  `}
+
+  &:hover {
+    background: ${props => props.theme.colors.greyBG};
+  }
+
+  ${is('expanded')`
+    color: ${props => props.theme.colors.white};
+    background: ${props => props.theme.colors.text};
+
+    &:hover {
+      background: ${props => props.theme.colors.text};
+    }
+  `}
+
+  ${is('darkTheme')`
+    &:hover {
+      background: #3A3553;
+    }
+
+    ${is('expanded')`
+      background: ${props => props.theme.colors.vibrant};
+      color: ${props => props.theme.colors.text};
+
+      &:hover {
+        background: ${props => props.theme.colors.vibrant};
+        color: ${props => props.theme.colors.textLight};
+      }
+    `}
+  `}
+`
+
+const DesktopMenuDropdownItems = styled.ul`
+  position: absolute;
+  width: ${remcalc(160)};
+  display: flex;
+  flex-direction: column;
+  top: ${remcalc(48)};
+  left: -9999px;
+  opacity: 0;
+  transition: opacity ${props => props.theme.animations.normal} ease;
+  background: ${props => props.theme.colors.greyBG};
+
+  ${is('expanded')`
+    left: 0;
+    opacity: 1;
+  `}
+
+  li {
+    display: flex;
+  }
+
+  a {
```

Comment:
I'm not 100% sure that I'm understanding how everything fits together & which styles are responsible for this. But I'm thinking that this combination of "vibrant" background colour + grey text on hover isn't ideal:
![image](https://user-images.githubusercontent.com/15656538/52807123-92e6ed00-3082-11e9-8a75-3c77c968aac4.png)

I think we want to make a distinction between the active, hover, focus states of the DropdownName & the sub-items?
Might it be worth putting these styles on the child directly, rather than by inheritance?

Reply:
Maybe you can ignore this. I've actually just seen this this state is documented explictly:
![image](https://user-images.githubusercontent.com/15656538/52862105-45bf5580-312c-11e9-8c49-5662aac61623.png)

So I reckon a discussion about colour contrast can wait until Antonas is back & we can just make it to specs for now